packages/itwinui-react/src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -233,11 +233,12 @@ const BreadcrumbsItem = React.forwardRef((props, forwardedRef) => {
   const defaultAs = !!props.href ? 'a' : !!props.onClick ? 'button' : 'span';
   const children =
     defaultAs === 'button' ? <span>{childrenProp}</span> : childrenProp;
+  const anchorClassName = defaultAs === 'a' ? 'iui-anchor' : '';
```

Comment:
`Anchor` component could be used here as well

Reply:
Will do.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/ecdd7cf46b7f29bc1f86b77f40b7f23364723ce5)

Reply:
You don't need to branch on the return value. I'm just suggesting this one line change:

```diff
- const defaultAs = !!props.href ? 'a' : !!props.onClick ? 'button' : 'span';
+ const defaultAs = !!props.href ? Anchor : !!props.onClick ? 'button' : 'span';
```

Reply:
I tried that as a string and it didn't work for me. Guess I didn't realize that it had to be a type if it is a React component. Will fix.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/4f086a8b25654918d9a640e46f5cc1ee1ba9735f)